src/catalog/catalog.cpp

```diff
@@ -202,6 +202,17 @@ void Catalog::dropTableSchema(transaction::Transaction* transaction, table_id_t
         // DO NOTHING.
     }
     }
+    for (auto& property : tableEntry->getPropertiesRef()) {
+        if (property.getDataType()->getLogicalTypeID() == LogicalTypeID::SERIAL) {
+            auto seqName = std::string(tableEntry->getName())
```

Comment:
Can u wrap this into a function? Should reuse it in `bind_Create_rdf_graph.cpp` too.